components/HomePage/9Products/index.jsx

```diff
@@ -127,15 +129,24 @@ const getProductList = (list, type) => list.map(eachProduct => {
           onClick={() => openUrl(primaryLink)}
           style={{ minWidth: '200px' }}
         />
-        {/* <Button
-          title="LEARN MORE"
-          type="black"
-          className="mini"
-          disabled
-          // TODO: disabled util the product pages are ready
-          // disabled={!secondaryLink}
-          // onClick={() => openUrl(secondaryLink)}
-        /> */}
+        {!isSecondaryLinkExternal ? (
+          <a href={`/${secondaryLink}`}>
+            <Button
+              title="LEARN MORE"
+              type="black"
+              className="mini"
+              disabled={!secondaryLink}
+            />
+          </a>
+        ) : (
+          <Button
+            title="LEARN MORE"
+            type="black"
+            className="mini"
+            disabled={!secondaryLink}
+            onClick={() => openUrl(secondaryLink)}
```

Comment:
This feels very convoluted to me – why aren't we just using href on the default antd button?